{
  "artifacts": [],
  "command": "do_experiment",
  "experiment": {
    "base_dir": "/home/enterprise.internal.city.ac.uk/acvn728/MScFinalProject",
    "dependencies": [
      "numpy==1.15.2",
      "sacred==0.7.4",
      "tensorflow-gpu==1.11.0"
    ],
    "mainfile": "main.py",
    "name": "UNet_Speech_Separation",
    "repositories": [],
    "sources": [
      [
        "audio_models.py",
        "_sources/audio_models_f18f61db39939ae190ffb11515e74407.py"
      ],
      [
        "dataset.py",
        "_sources/dataset_a83ad44a54b843a850f6c265244adb8b.py"
      ],
      [
        "main.py",
        "_sources/main_376e5599961f28a586ead94e099ea8a0.py"
      ],
      [
        "train.py",
        "_sources/train_a27355c1bba8ecc7c185198452fa55a2.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/tensorflow/python/ops/array_ops.py\", line 1439, in zeros\n    output = constant(zero, shape=shape, dtype=dtype, name=name)\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/tensorflow/python/framework/constant_op.py\", line 208, in constant\n    value, dtype=dtype, shape=shape, verify_shape=verify_shape))\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/tensorflow/python/framework/tensor_util.py\", line 380, in make_tensor_proto\n    if shape is not None and np.prod(shape, dtype=np.int64) == 0:\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/numpy/core/fromnumeric.py\", line 2585, in prod\n    initial=initial)\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/numpy/core/fromnumeric.py\", line 83, in _wrapreduction\n    return ufunc.reduce(obj, axis, dtype, out, **passkwargs)\n",
    "TypeError: __int__ returned non-int (type NoneType)\n",
    "\nDuring handling of the above exception, another exception occurred:\n\n",
    "Traceback (most recent call last):\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/sacred/config/captured_function.py\", line 46, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"main.py\", line 111, in do_experiment\n    name='Magnitude_Model')\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/MScFinalProject/audio_models.py\", line 34, in __init__\n    self.voice_mask_network = BasicCapsnet(mixed_mag, name='SegCaps_CapsNetBasic')\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/MScFinalProject/audio_models.py\", line 193, in __init__\n    routings=1, name='primarycaps')(net)\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/keras/engine/topology.py\", line 603, in __call__\n    output = self.call(inputs, **kwargs)\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/MScFinalProject/SegCaps/capsule_layers.py\", line 154, in call\n    num_routing=self.routings)\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/MScFinalProject/SegCaps/capsule_layers.py\", line 351, in update_routing\n    swap_memory=True)\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/tensorflow/python/ops/control_flow_ops.py\", line 2816, in while_loop\n    result = loop_context.BuildLoop(cond, body, loop_vars, shape_invariants)\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/tensorflow/python/ops/control_flow_ops.py\", line 2640, in BuildLoop\n    pred, body, original_loop_vars, loop_vars, shape_invariants)\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/tensorflow/python/ops/control_flow_ops.py\", line 2590, in _BuildLoop\n    body_result = body(*packed_vars_for_body)\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/MScFinalProject/SegCaps/capsule_layers.py\", line 332, in _body\n    activation = _squash(preactivate)\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/MScFinalProject/SegCaps/capsule_layers.py\", line 359, in _squash\n    return div_no_nan(input_tensor, norm) * div_no_nan(norm_squared, (1 + norm_squared))\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/MScFinalProject/SegCaps/capsule_layers.py\", line 370, in div_no_nan\n    res = tf.where(tf.is_nan(res), tf.zeros(res.shape), res)\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/tensorflow/python/ops/array_ops.py\", line 1441, in zeros\n    shape = ops.convert_to_tensor(shape, dtype=dtypes.int32, name=\"shape\")\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/tensorflow/python/framework/ops.py\", line 836, in convert_to_tensor\n    as_ref=False)\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/tensorflow/python/framework/ops.py\", line 926, in internal_convert_to_tensor\n    ret = conversion_func(value, dtype=dtype, name=name, as_ref=as_ref)\n",
    "  File \"/home/enterprise.internal.city.ac.uk/acvn728/.local/lib/python3.5/site-packages/tensorflow/python/framework/constant_op.py\", line 250, in _tensor_shape_tensor_conversion_function\n    \"Cannot convert a partially known TensorShape to a Tensor: %s\" % s)\n",
    "ValueError: Cannot convert a partially known TensorShape to a Tensor: (?, 128, 512, 8, 8)\n"
  ],
  "heartbeat": "2018-10-12T11:53:52.554949",
  "host": {
    "ENV": {},
    "cpu": "AMD Ryzen 7 1700 Eight-Core Processor",
    "gpus": {
      "driver_version": "396.44",
      "gpus": [
        {
          "model": "GeForce GTX 1080 Ti",
          "persistence_mode": false,
          "total_memory": 11177
        },
        {
          "model": "GeForce GTX 1080 Ti",
          "persistence_mode": false,
          "total_memory": 11178
        }
      ]
    },
    "hostname": "africa",
    "os": [
      "Linux",
      "Linux-4.4.0-137-generic-x86_64-with-Ubuntu-16.04-xenial"
    ],
    "python_version": "3.5.2"
  },
  "meta": {
    "command": "do_experiment",
    "options": {
      "--beat_interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print_config": false,
      "--priority": null,
      "--queue": false,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [],
      "help": false,
      "with": false
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2018-10-12T11:51:10.867675",
  "status": "FAILED",
  "stop_time": "2018-10-12T11:53:52.576536"
}